configurator/templates/AppServer/configs.ini

```diff
@@ -0,0 +1,28 @@
+[SETTINGS]
+read_env_variables=false
+server=AppServer
+pack_location=/Users/anuruddha/Downloads/wso2as-5.2.1/repository/conf
+
+[DEFAULTS]
+clustering=false
+members={"127.0.0.1": 4000,"127.0.1.1":4100}
+localMemberHost= 127.0.1.1
+localMemberPort= 4100
+subDomain= mgt
+stratos_instance_data_worker_host_name= as.wso2.com
+stratos_instance_data_mgt_host_name= mgt.as.wso2.com
```

Comment:
Any particular reason to have long key names here? (If this is configured by Puppet, the concern would be less).